libqtile/scripts/qtile_run.py

```diff
@@ -26,26 +26,39 @@
 import argparse
 import subprocess
 
-from libqtile import command
+from libqtile import command_graph, ipc
 
 
 def parse_args():
-    parser = argparse.ArgumentParser(description="Run a command applying rules"
-                                     " to the new windows")
-    parser.add_argument('-s', '--socket', type=str, dest="socket",
-                        help='Use specified communication socket.')
-    parser.add_argument('-i', '--intrusive',
-                        dest="intrusive", action='store_true',
-                        help='If the new window should be intrusive.')
-    parser.add_argument('-f', '--float', dest="floating", action='store_true',
-                        help='If the new window should be float.')
-    parser.add_argument('-b', '--dont-break', dest="dont_break",
-                        action='store_true',
-                        help='Don\'t break on match(keep applying rules).')
-    parser.add_argument('-g', '--group', dest="group", type=str,
-                        help='Set the window group.')
-    parser.add_argument('cmd', nargs=argparse.REMAINDER,
-                        help='Command to execute')
+    parser = argparse.ArgumentParser(
+        description="Run a command applying rules to the new windows")
+    parser.add_argument(
+        '-s',
+        '--socket',
+        help='Use specified communication socket.')
+    parser.add_argument(
+        '-i',
+        '--intrusive',
+        action='store_true',
+        help='If the new window should be intrusive.')
+    parser.add_argument(
+        '-f',
+        '--float',
+        action='store_true',
+        help='If the new window should be float.')
+    parser.add_argument(
+        '-b',
+        '--dont-break',
+        action='store_true',
+        help='Do not break on match (keep applying rules).')
+    parser.add_argument(
+        '-g',
+        '--group',
+        help='Set the window group.')
+    parser.add_argument(
+        'cmd',
+        nargs=argparse.REMAINDER,
+        help='Command to execute')
```

Comment:
disturbing indentation

Reply:
Haha, I picked this up as my typical indentation pattern after watching https://www.youtube.com/watch?v=ZsHMHukIlJY, but I'd be happy to change it back.

Reply:
I wouldn't mind this type of indentation if the closing parenthesis was on a new line. But to be honest my favorite one is just the one that was there before. :smile: 

And thank you Python for not enforcing an indentation so that we can keep having those passionate discussions. :heart: